utils/devel/sync-fork.sh

```diff
@@ -42,14 +44,15 @@ if ! [[ $AvailableRemotes =~ $Upstream ]]; then
     git remote add upstream "$Upstream"
 fi
 
-git fetch upstream || Error "Fetching Upstream."
-
+CurrentBranch=`git rev-parse --abbrev-ref HEAD`
 
 git checkout master || Error "Checking out Master."
 
+git fetch upstream || Error "Fetching Upstream."
 
 git merge upstream/master || Error "Merging upstream/master with local /master"
```

Comment:
Hi Erik,
    
    Well sort of.
    It merges into the active branch, but as we start out by checking
    out master, it effectively merges into master.
    
    We then checkout the original active branch and warn the user that
    they will need to do a merge.
    We can't really do the merge on the active branch automatically as
    it's difficult / impossible to know what branch they may want to
    base from.
    
    Regards
    David G
    
    On 04/05/16 04:54, Erik Huelsmann
      wrote:
    
    
      In utils/devel/sync-fork.sh:
      >  
>  git checkout master || Error "Checking out Master."
>  
> +git fetch upstream || Error "Fetching Upstream."
>  
>  git merge upstream/master || Error "Merging upstream/master with local /master"

      @sbts I think this merges into the
        active branch, not into the master branch, right?
      —
        You are receiving this because you were mentioned.
        Reply to this email directly or view
          it on GitHub